development/workflow.rst

```diff
@@ -0,0 +1,668 @@
+#######################################
+DM's Collaborative Development Workflow
+#######################################
+
+This page describes our procedures for collaborating on LSST DM software and documentation with `Git <http://git-scm.org>`_, `GitHub <https://github.com>`_ and `JIRA <https://jira.lsstcorp.org/>`_:
+
+1. :ref:`Configuring Git for DM development <git-setup>`.
+2. :ref:`Using JIRA for agile development <workflow-jira>`.
+3. :ref:`Policies for naming and using Git branches <git-branching>`.
+4. :ref:`Preparing code for review <review-preparation>`.
+5. :ref:`Reviewing and merging code <workflow-code-review>`.
+
+In appendices, we suggest some *best practices* for efficient collaborative development:
+
+- :ref:`Git and GitHub configuration suggestions <git-setup-best-practices>`.
+- :ref:`Commit organization best practices <git-commit-organization-best-practices>`.
+- :ref:`Commit message best practices <git-commit-message-best-practices>`.
+- :ref:`Tips for using JIRA <workflow-jira-tips>`.
+
+You can also read the :doc:`lsstsw Stack Development Tutorial </development/lsstsw_tutorial>` to see how our processes work in a holistic stack development workflow.
+
+.. _git-setup:
+
+Git & GitHub Setup
+==================
+
+You need to install Git version 1.8.2, or later, and the Git LFS client to work with our data repositories.
+See the :doc:`DM Git LFS documentation </development/git_lfs>` for more information on how to install and setup the Git LFS client for your machine.
+
+We use Git commit authorship metadata to audit copyrights in DM code.
+Ensure that Git is setup to use your *institution-hosted* email address (only AURA employees should use their ``lsst.org`` email addresses) in the :file:`~/.gitconfig` file.
+You can do this from the command line:
+
+.. code-block:: bash
+
+   git config --global user.name "Your Name"
+   git config --global user.email "your_email@institution.edu"
+
+Likewise, in your `GitHub account email settings <https://github.com/settings/emails>`_, add your institution-hosted email.
+We recommend that you set this institutional email as your **Primary GitHub** email address.
+This step ensures that Git commits you make `directly on GitHub.com <https://help.github.com/articles/github-flow-in-the-browser/>`_ (such as quick documentation fixes) have proper authorship metadata.
```

Comment:
Problematic for me as well. I really wish Github would do the right thing. They already send all these messages to the correct email address based on the organization so it amazes me they can't use that same mapping for the commits.